### PR TITLE
add something 555

### PR DESCRIPTION
**Pull request by prompath**
accept please

### CODE REVIEW

**Comment by prompath** [comment]
😊😊😊😊😊😊😊🐼🐼🐼🐼🐼🐼📈📈📈📈🐈🐈🐈🐈🐈

File: src/pythonStarter/hello_world.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 def print_message(msg):
+
     print(msg + '!')
+    raise ValueError("Wrong Answer!")
```
Comment (WinEisEis) [inline src/pythonStarter/hello_world.py:4]: What an ugly code!

File: src/pythonStarter/hello_world.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 def print_message(msg):
+
```
Comment (WinEisEis) [inline src/pythonStarter/hello_world.py:2]: 1st review

**Review by WinEisEis — COMMENTED** [review]
1st revision

File: src/pythonStarter/hello_world.py (hunk 1 above)
Comment (prompath) [reply src/pythonStarter/hello_world.py:4]: 😢

Comment (prompath) [reply src/pythonStarter/hello_world.py:4]: I'm quitting this company!

File: src/pythonStarter/hello_world.py (hunk 2 above)
Comment (prompath) [reply src/pythonStarter/hello_world.py:2]: please I coded this for so long

**Comment by WinEisEis** [comment]
I would like to reject your commit.

**Comment by WinEisEis** [comment]
2nd rejection

**Pull request closed without merging**